### DISCUSSION TITLE
跨域问题及解决方案

### DISCUSSION THREAD

**Discussion by giscus** [ztkuaikuai/blog#14] [General]
# 跨域问题及解决方案

记录一些技术、生活、阅读的感悟；是我的人类使用说明书。

https://blog.kuaikuaitz.top/posts/%E8%B7%A8%E5%9F%9F%E9%97%AE%E9%A2%98%E5%8F%8A%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88

**Comment by ztkuaikuai** [comment]
测试第三方 cookie
